test/lib.bash

```diff
@@ -170,6 +180,51 @@ function end_prober_test {
   wait ${PROBER_PID}
 }
 
+function run_knative_serving_operator_tests {
+  (
+  local version target serverless_rootdir exitstatus patchfile fork gitdesc
+  version="$1"
+  fork="${2:-knative}"
+  serverless_rootdir="$(pwd)"
+  make_temporary_gopath
+
+  logger.info "Checkout the code ${fork}/serving-operator @ ${version}"
+  mkdir -p "${GOPATH}/src/knative.dev"
```

Comment:
What about this?:
```
target="${GOPATH}/src/knative.dev/serving-operator"
mkdir -p $target
git clone .... "$target"
```
This would remove a bit of redundancy.

Reply:
Sure.

Reply:
Let's fix it in the other PR. and get this in